### ISSUE TITLE
v2.6.1 is incompatible with gem thinking-sphinx

### ISSUE THREAD

**Issue by aepyornis** [bensheldon/good_job#450]
The [search scope](https://github.com/bensheldon/good_job/pull/440/files#diff-5faa8df5a47008de9d75ee5c323bc397cce04c2dce0c797a95fa5f5c2daa35eeR32) added by #440 will raise an error if ActiveRecord::Base already has defined search, which happens when you use the gem [thinking-sphinx](https://github.com/pat/thinking-sphinx)

I can reproduce the error in a new project like this:
``` sh
rails new . --skip-javascript --database=postgresql --skip-spring
echo "gem 'mysql2'" >> Gemfile
echo "gem 'thinking-sphinx'" >> Gemfile
echo "gem 'good_job', '2.6.1'" >> Gemfile
bundle install
# setup config/database.yml for your postgres
bundle exec rails db:reset
bundle exec rails g good_job:install
bundle exec rails db:migrate
bundle exec rails runner "GoodJob::Execution"
```

The error raised:
```
~/.rbenv/versions/3.0.2/lib/ruby/gems/3.0.0/gems/activerecord-6.1.4.1/lib/active_record/scoping/named.rb:160:in `scope': You tried to define a scope named "search" on the model "GoodJob::Execution", but Active Record already defined a class method with the same name. (ArgumentError)
```

**Comment by bensheldon** [comment]
@aepyornis Thanks for the bug report! 🙏  I have a fix I will release momentarily.

**Comment by bensheldon** [comment]
I renamed the method to`#search_text` and released in [`2.6.2`](https://github.com/bensheldon/good_job/releases/tag/v2.6.2).

**Comment by aepyornis** [comment]
thank you!
